_data/3.7-drivers.yml

```diff
@@ -55,6 +55,13 @@
       href: go-connection-management.html
     - text: Reference
       href: https://godoc.org/github.com/arangodb/go-driver
+- text: ArangoDB C#/.NET Driver
+  href: net.html
```

Comment:
The files should be named dotnet* instead of net* in my opinion. https://en.wikipedia.org/wiki/.NET